checker/tests/signature/SignatureTypeFactoryTest.java

```diff
@@ -215,7 +215,7 @@ void m() {
         us = s12;
         // :: error: (assignment.type.incompatible)
         fqn = s12;
-        bn = s12;
+        // bn = s12; // s12 isn't a valid BinaryName
```

Comment:
Please don't comment out tests.  If an error is expected, write a `// ::` comment on the preceding line.